### PR TITLE
[Docs] New doc for keys and identities

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Ready for review and merge. 

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4755)

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
Docs preview link: https://deploy-preview-4755--aptos-developer-docs.netlify.app/nodes/identity-and-configuration

File: developer-docs-site/docs/nodes/identity-and-configuration.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+### public-keys.yaml
+
+#### Example
+
+Click below to see an example YAML configuration:
+<details>
+<summary>public-keys.yaml</summary>
+
+```yaml
+---
```
Comment (movekevin) [inline developer-docs-site/docs/nodes/identity-and-configuration.md:90]: Do we want to note that this is the validator address (stake pool address or machine address whatever you want to consistently call it), and not the owner, operator, or voter address? cc @sherry-x. We can also link to the get-stake-pool CLI doc to explain how to get all of these addresses (from owner address) as well

File: developer-docs-site/docs/nodes/identity-and-configuration.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+```bash
+aptos genesis generate-keys --output-dir ~/$WORKSPACE/keys
+```
+
+See below a diagram that shows how the identities for the validator node and validator fullnode are derived from the private and public keys:
+
+<ThemedImage
+alt="Signed Transaction Flow"
+sources={{
+    light: useBaseUrl('/img/docs/key-yamls.svg'),
```
Comment (movekevin) [inline developer-docs-site/docs/nodes/identity-and-configuration.md:76]: Is this an existing image? I don't see it included in this PR

**Review by movekevin — APPROVED** [review]

File: developer-docs-site/docs/nodes/identity-and-configuration.md (hunk 2 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/identity-and-configuration.md:76]: Haha, I thought the same for a minute. I had some merge conflicts and not sure how I ended up merging this figure before. I have a few PRs in flight, all potentially leading to a merge conflict, so after all PRs land, I will figure this out.

File: developer-docs-site/docs/nodes/identity-and-configuration.md (hunk 1 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/identity-and-configuration.md:90]: Yes. This is exactly the kind of review feedback I needed. I didn't understand how I should understand the account address here. With the above explanation, @movekevin, now I have a different question: how is this address different from a resource account address? I won't mention it, but I want to understand it. Yes to linking also.

File: developer-docs-site/docs/nodes/identity-and-configuration.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+# Identity and Configuration
+
+When installing a node on an Aptos network, the installation steps require you to work with identities and configurations. This document describes how to interpret the terms **identity** and **configuration**, and presents a description of the identity YAML files.
+
+## Concept
+
+This section presents a mental-model view of an identity and configuraion. It is meant to help make the node installation process easy.
```
Comment (alinush) [inline developer-docs-site/docs/nodes/identity-and-configuration.md:16]: Typo: Confi**g**uration

**Review by gregnazario — APPROVED** [review]

**Pull request merged**